src/main/java/edu/harvard/iq/dataverse/util/FileUtil.java

```diff
@@ -86,6 +86,16 @@
     private static final Logger logger = Logger.getLogger(FileUtil.class.getCanonicalName());
     
     private static final String[] TABULAR_DATA_FORMAT_SET = {"POR", "SAV", "DTA", "RDA"};
+    // The list of formats for which we need to re-test, if we want to attempt 
+    // to ingest the file again:
+    // Example: We have added support for Stata-13+; the new ingest plugin is called
+    // when the file type is detected as "application/x-stata-1[345]". But the 
+    // previously uploaded, but not ingested stata files are in the database
+    // typed as "application/x-stata". So we want to run the new-and-improved
+    // DTA check that will re-identify the specific DTA flavor. 
+    // If similar cases are introduced in the future, the affected formats will
+    // need to be added to the list. 
```

Comment:
Why not just retest all? it's not expensive (I think).

Reply:
OK, I think you're right on this. It could actually be useful to retest everything; in case the type detection code has been improved since the last ingest attempt.